tests/phpunit/includes/TestCase.php

```diff
@@ -29,7 +28,13 @@ public static function setUpBeforeClass() {
 		parent::setUpBeforeClass();
 
 		if ( ! self::$featureFlagsConfig ) {
-			self::$featureFlagsConfig = new JSON_File( GOOGLESITEKIT_PLUGIN_DIR_PATH . 'feature-flags.json' );
+			self::$featureFlagsConfig = array();
+			if ( file_exists( GOOGLESITEKIT_PLUGIN_DIR_PATH . 'feature-flags.json' ) ) {
```

Comment:
We don't need to check if this file exists because it's under version control (it's only not present in the built zip, but neither are tests).

Reply:
In tests, this would create some errors if it does not exist as the wrapper `JSON_File` was doing that for us if we include it tests would fail due to the warnings.

Reply:
I tried reproducing this in the tests but wasn't able to. Can you show me the failure you're getting without this?

As an aside, I think most of this flag resetting here is unnecessary as this array only sets the list of available flags now which doesn't have any affect on feature activation (except that it requires the list to be non-empty). We can probably actually remove most of this but let's not do that just yet.

Reply:
This was a mistake on my end by confusing this file with the file from the main plugin file.

I just updated it by removing this conditional. Thanks for the feedback and follow-up patch, that we can introduce in the upcoming ticket.